TemplateBicep/modules/function.bicep

```diff
@@ -0,0 +1,174 @@
+@description('Provision a final LoRa device in the IoT hub in addition to the gateway.')
+param deployDevice bool
+
+@description('The Git Username. Default is Azure.')
+param gitUsername string = 'Azure'
+
+@description('The Azure location where to create the hosting plan. Default value is resource group location.')
+param hostingPlanLocation string = resourceGroup().location
+
+@description('Hosting plan SKU. Default is Y1.')
+param hostingPlanSkuName string = 'Y1'
+
+@description('Hosting Plan SKU tiers. Default value is Dynamic.')
+param hostingPlanSkuTier string = 'Dynamic'
+
+@description('Prefix used for resource names. Should be unique as this will also be used for domain names.')
+param uniqueSolutionPrefix string
+
+@description('Controls whether observability is set up for IoT Edge.')
+param useAzureMonitorOnEdge bool
+
+@description('The Git version to use. Default is 2.1.0.')
```

Comment:
This param should be propagated to the main. Bicep (we need to change it at every release, would be nice to have a central location)